### PR TITLE
Update Angle limits

### PR DESCRIPTION
**Pull request by carlos-m159**
If no reconfiguration was requested, the FOV was never set according to the configuration: 
https://github.com/locusrobotics/urg_node/blob/5f830173e6482a852497ec08bb96b957e16795fe/include/uam/uam_driver_ros.h#L126

### CODE REVIEW

File: src/uam/uam_driver_ros.cpp [hunk 1]
```diff
@@ -210,7 +210,14 @@ bool UamROS::configure()
 
     // TODO(cribeiromendes): make scip commands work seamlessly. Right now we
     // need to ask this before starting continuous async reads
+
+    // At this point the receiver thread is not yet running, so we can safely
+    // write into scan_params
     scan_params_ = lidar_.getScanDetails();
+    // Since the spinner is running this method and is also the responsible to
```
Comment (svwilliams) [inline src/uam/uam_driver_ros.cpp]:
```suggestion
    // Since the spinner is running this method and is also responsible to
```

**Review by svwilliams — APPROVED** [review]

**Comment by carlos-m159** [comment]
I am sorry, found out about some other issues. Will be updating this PR. Ignore for now

File: include/uam/uam_driver_ros.h [hunk 2]
```diff
@@ -152,8 +148,51 @@ class UamROS
...
+
+    const auto number_of_readings = last_step - first_step + 1;
+
+    if (reply.ranges.size() < number_of_readings)
+    {
+      ROS_ERROR_STREAM(
+        "Unexpected outcome: " << reply.ranges.size() << ", first and last_step are " << first_step << ", "
+                               << last_step);
+      return;
+    }
```
Comment (svwilliams) [inline include/uam/uam_driver_ros.h]: In the original code, you had the check:
```
  if (reply.ranges.size() <= last_step)
  {
    ROS_ERROR_STREAM("Unexpected outcome: " << reply.ranges.size() << " and last_step is " << last_step);
    return;
  }
```
That check seems more correct to me. The `reply.ranges` container appears to be the source container that has all of the range readings, not just the requested subset. Therefore the size of `reply.ranges` must be greater than `last_step` and not merely greater than the size of the requested subset.

**Review by svwilliams — APPROVED** [review]

**Comment by carlos-m159** [comment]
Rebased to release/23

**Review by ShivamThukral — APPROVED** [review]

**Pull request merged**